### PR TITLE
Add docs for extensions

### PR DESCRIPTION
**Pull request by Kidswiss**

### CODE REVIEW

**Review by lieneluksika — APPROVED** [review]

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Add extensions to the instance
+
+PostgreSQL supports many different extenions, such as PostGIS and TimescaleDB.
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:3]:
```suggestion
PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] and https://www.timescale.com/[TimescaleDB^].
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+These extensions can be used in PostgreSQL by VSHN as well.
+For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page].
+
+== Enable extenions
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:7]:
```suggestion
== Enable PostgreSQL Extensions
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To enable an extension add the following to your instance:
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:9]:
```suggestion
To enable an extension, add the following configuration to your instance:
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+metadata:
+  name: pgsql-app1-prod
+spec:
+  parameters:
+    service:
+      extensions:
+        - name: ltree
+----
+
+After applying the change AppCat will ensure that the extension is available after a short while.
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:24]:
```suggestion
After applying the change, AppCat will ensure that the extension is available after a short while.
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+WARNING: Some extension like `timescaldb` or `postgis` will require a restart of the instance to be available.
+Depending on your https://docs.appcat.ch/vshn-managed/postgresql/update-strategy.html[Instance Update Strategy] it's possible that the instance will restart immediately.
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:27]:
```suggestion
Depending on your https://docs.appcat.ch/vshn-managed/postgresql/update-strategy.html[Instance Update Strategy], it's possible that the instance will restart immediately.
```

**Review by tobru — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
+= Add extensions to the instance
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:1]:
```suggestion
= PostgreSQL Extensions
```

As the title appears on the navigation, I'd pick a shorter title here.

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] and https://www.timescale.com/[TimescaleDB^].
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:3]:
```suggestion
PostgreSQL supports many different extensions, such as https://postgis.net/[PostGIS^] or https://www.timescale.com/[TimescaleDB^].
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+These extensions can be used in PostgreSQL by VSHN as well.
+For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page].
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:5]:
```suggestion
For a list of all available extensions please https://stackgres.io/doc/latest/intro/extensions/[consult this page^].
```

Let's open external links in a new tab

File: docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+kind: VSHNPostgreSQL
+metadata:
+  name: pgsql-app1-prod
+spec:
+  parameters:
+    service:
+      extensions:
+        - name: ltree
+----
+
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/extensions.adoc:23]:
```suggestion
        - name: ltree <1>
----

<1> Name of the extension as found on the "Name" column at https://stackgres.io/doc/latest/intro/extensions/[the official listing^]. Add more extensions to the list as needed.
```

**Pull request merged**